tools/cmake/app.cmake

```diff
@@ -12,11 +12,11 @@ include(${MBED_CONFIG_PATH}/mbed_config.cmake)
 # Load toolchain file
 if(NOT CMAKE_TOOLCHAIN_FILE OR MBED_TOOLCHAIN_FILE_USED)
     set(MBED_TOOLCHAIN_FILE_USED TRUE CACHE INTERNAL "")
-    include(${mbed-os_SOURCE_DIR}/tools/cmake/toolchain.cmake)
+    include(${CMAKE_CURRENT_LIST_DIR}/toolchain.cmake)
```

Comment:
OK, I see you changed these in a later commit. It would be easier to review if we didn't make changes to already reviewed commits later on in the PR.

Reply:
I do not want to push force but I can clean it up.

Reply:
> It would be easier to review if we didn't make changes to already reviewed commits later on in the PR.

I do not follow. I fixed earlier commit in the subsequent one. how else can I make review easier without rebasing?

Reply:
I usually would force push, but marking them as `fixup` commits would also work for me.

Reply:
+1 for force push but that is hard for some reviewers so I can mark as fixu and later rebase to get them out of history.